jorge/js/app.js

```diff
@@ -1,17 +1,40 @@
 //Variables
-const register = document.getElementById('register')
-const login = document.querySelector('#login')
+const registerBtn = document.getElementById('register')
+const loginBtn = document.querySelector('#login')
+const registerForm = document.getElementById('registerForm')
+
+
 
 // addEventListeners
 
-register.addEventListener('click', showRegisterForm)
+registerBtn.addEventListener('click', showRegisterForm)
 
-login.addEventListener('click', showLoginForm)
+loginBtn.addEventListener('click', showLoginForm)
+
+registerForm.addEventListener('submit', submitRegister)
 
 // Functions
 
 function showRegisterForm () {
   console.log('SHOW REGISTER')
+  registerForm.style.display = 'flex'
+}
+
+function submitRegister (e) {
+  e.preventDefault()
+  
+  const form = this.firstElementChild
```

Comment:
NIT: I would recommend to create the element reference variables at the top of the js file in case we need to reuse them. Even though this code works, I would recommend using querySelector because is easier to read. Whenever you add a `this.` you add complexity to the code because I need to double check which is the context this function is being called from so I know for sure what ill be storing in this variable, but since querySelector works on a document level you make it easier for me to be sure Ill always have the form stored in this variable without worrying if I got it from the right context.